app/warnings_code/update_warning_check_task_status.sql

```diff
@@ -0,0 +1,23 @@
+-- stored procedure to resume or suspend check_warnings_every_minute task
```

Comment:
Ditto on missing grant permissions for all `warnings_code` files. Are those not needed in order for the app to call them?

Reply:
permissions were never included in the original app. @sfc-gh-cgorrie do you have any more info on that?

Reply:
Yes, it's because streamlits run as `app_primary`, which is the owner of anything created in the setup script and thus does not require an explicit grant statement. Anything only called from streamlits doesn't need grant statements (surprisingly)